src/path_string.rs

```diff
@@ -31,7 +32,17 @@ impl PathString for EvaluationContext<'_> {
         "args" => ctx.args()?.get_path(tail).cloned(),
         "headers" => ctx.get_header_as_value(&tail[0]),
         "vars" => Some(async_graphql::Value::String(
-          ctx.req_ctx.server.vars.clone()?.get(&tail[0]).cloned()?,
+          ctx
+            .req_ctx
+            .server
+            .vars
+            .clone()
+            .map(|x| {
+              let b: BTreeMap<String, String> = x.into();
+              b
+            })?
+            .get(&tail[0])
+            .cloned()?,
```

Comment:
This will slow down our existing implementation. Can we consider optimizing it or atleast match the performance from main.
- [ ] Post wrk benchmarks before and after this change as a PR.